lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -367,7 +368,12 @@ def add_stored_credential(xml, options)
 
       def merchant_initiated_unscheduled(options)
         return options[:merchant_initiated_unscheduled] if options[:merchant_initiated_unscheduled]
-        return 'Y' if options.dig(:stored_credential, :initiator) == 'merchant' && options.dig(:stored_credential, :reason_type) == 'unscheduled'
+        return 'Y' if options.dig(:stored_credential, :initiator) == 'merchant' && options.dig(:stored_credential, :reason_type) == 'unscheduled' || options.dig(:stored_credential, :reason_type) == 'recurring'
```

Comment:
This looks right, but it seems unrelated to the `ssl_add_token` field. Just want to make sure this should be changed.

Reply:
Since recurring transactions always need the merchant_intiated_unscheduled field but only the initial recurring transaction needs ssl_add_token, upon further reflection it made more sense for that check to live here.